libz-rs-sys/src/lib.rs

```diff
@@ -56,31 +65,6 @@ macro_rules! prefix {
 const _: () =
     compile_error!("Only one of `rust-allocator` and `c-allocator` can be enabled at a time");
 
-#[allow(unreachable_code)]
-const DEFAULT_ZALLOC: Option<alloc_func> = '_blk: {
-    // this `break 'blk'` construction exists to generate just one compile error and not other
-    // warnings when multiple allocators are configured.
-
-    #[cfg(feature = "c-allocator")]
-    break '_blk Some(zlib_rs::allocate::Allocator::C.zalloc);
-
-    #[cfg(feature = "rust-allocator")]
-    break '_blk Some(zlib_rs::allocate::Allocator::RUST.zalloc);
-
-    None
-};
-
-#[allow(unreachable_code)]
-const DEFAULT_ZFREE: Option<free_func> = '_blk: {
-    #[cfg(feature = "c-allocator")]
-    break '_blk Some(zlib_rs::allocate::Allocator::C.zfree);
-
-    #[cfg(feature = "rust-allocator")]
-    break '_blk Some(zlib_rs::allocate::Allocator::RUST.zfree);
-
-    None
-};
```

Comment:
this is handled by zlib-rs. Only one allocator is available there because of the compile_error on line 66